### PR TITLE
Bump Citus Enterprise version to 9.5.1 & Enable PG13

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+  * Fixes a bug that could cause excessive memory consumption when a partition is
+  created
+
+  * Fixes a bug that triggers subplan executions unnecessarily with cursors
+
+  * Fixes a segfault in connection management due to invalid connection hash
+  entries
+
+ -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Wed, 2 Dec 2020 14:29:1 +0000
```
Comment (onurctirtir) [inline debian/changelog]:
```suggestion
 -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Wed, 2 Dec 2020 14:29:10 +0000
```

This should fix ci warnings (cc @hanefi )

Comment (hanefi) [inline debian/changelog]:
```suggestion
 -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Wed, 2 Dec 2020 14:29:01 +0000
```

**Review by hanefi — APPROVED** [review]
There seems to be a new warning. It should not be an issue I guess

EL7 test https://travis-ci.com/github/citusdata/packaging/jobs/453587565

```
/bin/sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory
sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory
sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory/bin/sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory
sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory
sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory
```

**Comment by onurctirtir** [comment]
> There seems to be a new warning. It should not be an issue I guess
> 
> EL7 test https://travis-ci.com/github/citusdata/packaging/jobs/453587565
> 
> ```
> /bin/sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory
> sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory
> sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory/bin/sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory
> sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory
> sh: warning: setlocale: LC_ALL: cannot change locale (C.utf8): No such file or directory
> ```

Let me test the package to make sure that package is fine, could be related with https://github.com/citusdata/packaging/pull/513/

edit: pg 13 package looks fine, I opened an issue to fix that later
https://github.com/citusdata/packaging/issues/572

**Comment by onurctirtir** [comment]
There was a compiler warning, opened a seperate issue for that as well https://github.com/citusdata/citus-enterprise/issues/516

**Pull request merged**